src/adjoint/masterRoutines.F90

```diff
@@ -144,6 +158,7 @@ subroutine master(useSpatial, famLists, funcValues, forces, &
     do sps=1, nTimeIntervalsSpectral
        do nn=1, nDom
           call setPointers(nn, 1, sps)
+          call timeStep_block(.false.)
```

Comment:
Why did you move this call here? We would need to also adjust the master_d and master_b routines to reflect the correct order because they are hand-differentiated.

Reply:
ok, the order is consistent with others. I am curious why this was not updated here before. 

I think I also remembered why we were struggling with this before; when we did the blockette stuff, we weren't putting back the radI/J/K variables from blockette memory, which was causing issues with other residual evaluations.

either way, the change in this PR looks fine.